### ISSUE TITLE
Add support for 'bridge' link type in ip_link_device.py

### ISSUE THREAD

**Issue by vanyasvl** [amarao/collection_ip#1]
Add support for 'bridge' link type in ip_link_device.py

Example:
```
ip link add bridge0 type bridge vlan_filtering 1
brctl show

root@vm1:~# brctl show
bridge name	bridge id		STP enabled	interfaces
bridge0		8000.000000000000	no
```

**Comment by amarao** [comment]
Hello. Thank you for feedback. Can you write an integration test for things you need?

**Comment by amarao** [comment]
Should be solved in version 0.1.4
